Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: build
+build: go-modules-tidy
+	go build -o bin/felix cmd/felix/main.go
+
+.PHONY: go-modules-tidy
+go-modules-tidy:
```

Comment:
I don't think you'll ever want to tidy without vendor, so for the grpc template I had one make task for vendor.
```
vendor:
	go mod tidy
	go mod vendor
```